src/instructions.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Make a `Digital-Twins/instructions` folder separate from the `Digital-Twins/src` folder to separate instructions form code